integration/spadev0/util.go

```diff
@@ -90,9 +90,9 @@ func AddSpadeTasks(ctx context.Context, requester string, replicasToTest map[int
 }
 
 var spadev0Metadata map[string]string = map[string]string{
-	"retrieve_type": "spade",
-	"retrieve_size": "1048576",
-	// todo: specify # of cids to test per layer of the tree TBD
+	"retrieve_type":  string(task.Spade),
+	"retrieve_size":  "1048576",
+	"traverse_depth": "3",
```

Comment:
Is this always three or should it be configurable

Reply:
All of the spade retrieval tests will have 3 for now, since they're following the [dagaggregator](https://github.com/filecoin-project/go-dagaggregator-unixfs#grouping-unixfs-structure) model. We could parameterize in the future if we need but I think it's okay to hardcode it for now 🤔

Reply:
Wait, no, this is not true. The daggregator model is a really dumb legacy thing that I wrote, and only the legacy daghaus uploads are using it. We are looking to completely stop using this by the end of this year.

So while ok to ship like this, expecting all dags to be of this form is definitely wrong.

Reply:
Ok, will pull it out into an environment variable